src/ui/api/config.go

```diff
@@ -338,11 +338,7 @@ func convertForGet(cfg map[string]interface{}) (map[string]*value, error) {
 		return nil, err
 	}
 	//All configuration of UAA will be un-editable for PKS 1.0 (1.4)
```

Comment:
CI failed and the UAA un-editable comment should be updated as well.

Reply:
@jessehu Thanks the misleading comment will be removed, the CI failure maybe due to slow render, and the code failed to get the element, we'll see